mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -127,6 +135,10 @@ CLASS none/sg net/minecraft/entity/Entity
 	METHOD a damage (Lnone/rr;F)Z
 		ARG 0 source
 		ARG 1 damage
+	METHOD a (Lnone/sg;)V
+	METHOD a startRiding (Lnone/sg;Z)Z
```

Comment:
What about the hypothetical scenario of "riding" a stationary object? Like a chair. In such a case I'd find `mount` more fitting. Perhaps we could find an entirely different name for it, like... I dunno `attach`?